core/sql/exp/ExpLOB.cpp

```diff
@@ -6,7 +6,9 @@
 // distributed with this work for additional information
 // regarding copyright ownership.  The ASF licenses this file
 // to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
+// "License
+
+"); you may not use this file except in compliance
```

Comment:
Will this compile? Looks like a stray line feed got added to the code.

Reply:
It compiled since it was  encompassed in a larger comment but correcting it anyway.